docs/conf.py

```diff
@@ -13,6 +13,8 @@
 
 import sys, os
 
+sys.path.insert(0, os.path.dirname(os.path.dirname(__file__)))
```

Comment:
I usually use `..` with `normpath`:

```python
>>> normpath(join('a/b/c', '../..'))
'a'
```

and if we are in a git repos, I will use this trick `git rev-parse --show-toplevel 2> /dev/null`.